samples/wifi/sta/src/main.c

```diff
@@ -319,7 +319,11 @@ void main(void)
 		}
 		LOG_DBG("\n");
 
-		qspi_enable_encryption(key);
+		ret = qspi_enable_encryption(key);
+		if (ret) {
+			LOG_ERR("Failed to enable encryption: %d\n", ret);
+			return;
+		}
```

Comment:
Maybe better to have a INFO message saying "QSPI encryption enabled" in else case here?

Reply:
There is one above, I will just move it here.